system-x/services/hawtio/src/main/java/software/tnb/hawtio/client/local/LocalHawtioClient.java

```diff
@@ -0,0 +1,65 @@
+package software.tnb.hawtio.client.local;
+
+import software.tnb.common.utils.HTTPUtils;
+import software.tnb.hawtio.client.HawtioClient;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class LocalHawtioClient implements HawtioClient {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LocalHawtioClient.class);
+
+    private final String baseUrl;
+    private final OkHttpClient client;
```

Comment:
why not using HTTPUtils directly?

Reply:
the client for local instance is already `HTTPUtils.trustAllSslClient()` but I needed to create the interface  `software.tnb.hawtio.client.HawtioClient` because of the different implementation of the client on OCP (due to authenticated api)

Reply:
I meant why using `OkHttpClient` and not `HTTPUtils` to send the requests since it seems you're just sending standard get and post requests

Reply:
ok I try to replace it

Reply:
done, you are right, a little bit less code thanks